FairmaticSDK.xcframework/ios-arm64/FairmaticSDK.framework/Headers/FairmaticSDK-Swift.h

```diff
@@ -1054,7 +1054,9 @@ SWIFT_CLASS_PROPERTY(@property (nonatomic, class, readonly) BOOL isAccidentDetec
 /// A valid <code>Settings</code> object if the SDK is setup, otherwise nil.
 + (FairmaticSettings * _Nullable)getSettings SWIFT_WARN_UNUSED_RESULT;
 /// Logs all the critical permissions required by the SDK
-/// \param sdkHealthReason indicates the reason for logging SDK Health
+/// \param reason Indicates the reason for logging SDK Health
+///
```

Comment:
NIT: Removing empty line?

Reply:
This header is autogenerated by Xcode, hence cannot edit.